test/controllers/v1/rules_controller_test.rb

```diff
@@ -5,80 +5,138 @@
 module V1
   class RulesControllerTest < ActionDispatch::IntegrationTest
     setup do
-      RulesController.any_instance.stubs(:authenticate_user).yields
       User.current = FactoryBot.create(:user)
       @profile = FactoryBot.create(:profile, :with_rules)
     end
 
-    test 'index lists all rules' do
```

Comment:
I encourage you to look at this diff with whitespace changes removed. I placed the existing tests in a context which stubs out authentication.